tests/periph_wdg/tests/01-run.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2019 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+import json
+from testrunner import run
+
+# We test only up to 10ms, with smaller times mcu doesn't have time to
+# print sysyem time before resetting
```

Comment:
typo: should be sys**t**em instead of sys**y**em

Reply:
fixed